File: C#/BluffinMuffin.Protocol/Game/BetTurnStartedCommand.cs
using System;
using BluffinMuffin.Protocol.DataTypes.Attributes;
using BluffinMuffin.Protocol.DataTypes.Enums;

namespace BluffinMuffin.Protocol.Game
{
    /// <summary>
    /// This command is issued by the server when a betting round is starting. For example, when the 3 first cards are shown, it's the beginning of the flop.
    /// </summary>
    public class BetTurnStartedCommand : AbstractGameCommand
    {
        /// <summary>
        /// OBSOLETE: The round that is starting
        /// </summary>
        [ExampleValue(RoundTypeEnum.Flop)]
        [Obsolete("Use BettingRoundId instead")]
        public RoundTypeEnum Round { get; set; }
        /// <summary>
        /// The id of the betting round that is starting, starting at 1. For texas hold'em, Preflop=1, Flop=2, Turn=3, River=4
        /// </summary>
        [ExampleValue(1)]
        public int BettingRoundId { get; set; }
        /// <summary>
        /// All the card ids currently visible on the board (Preflop: empty, flop: 3 cards, turn: 4 cards, river: 5 cards) 
        /// </summary>
        [ExampleValues(3, "2s", "Kh", "5d")]
        public string[] Cards { get; set; }
    }
}

File: Python/BluffinMuffin.Protocol/game/bet_turn_started_command.py
from abstract_game_command import AbstractGameCommand
from data_types.enums.round_type_enum import RoundTypeEnum

__author__ = 'ericmas001@gmail.com'



class BetTurnStartedCommand(AbstractGameCommand):
    def __init__(self, obj):
        super().__init__(obj)
        self.round = RoundTypeEnum.parse(obj['Round'])
        self.betting_round_id = obj['BettingRoundId']
        self.cards = obj['Cards']

    def __str__( self ):
        return '{0} ({1}:{2} [{3}])'.format(super().__str__(),self.betting_round_id, RoundTypeEnum.to_string(self.round), ', '.join(self.cards))
